factory/api/export_test.go

```diff
@@ -27,7 +27,7 @@ type SCQueryElementArgs struct {
 	WorkingDir            string
 	Index                 int
 	GuardedAccountHandler process.GuardedAccountHandler
-	BlockChainHookCreator hooks.BlockChainHookHandlerCreator
+	ChainRunType          common.ChainRunType
```

Comment:
Is it ok that the factory was removed and replaced with this?

Reply:
I see this was a mistake, but this part was refactored again in the second PR and `BlockChainHookCreator` was replaced by these 2 [factories](https://github.com/multiversx/mx-chain-go/blob/5a76db36f0b8e61ce13f240115c90208e1bb0f5d/factory/api/export_test.go#L30)